scalastyle-config.xml

```diff
@@ -3,7 +3,7 @@
  <check level="error" class="org.scalastyle.file.FileTabChecker" enabled="true"></check>
  <check level="error" class="org.scalastyle.file.FileLengthChecker" enabled="true">
   <parameters>
-   <parameter name="maxFileLength"><![CDATA[500]]></parameter>
+   <parameter name="maxFileLength"><![CDATA[525]]></parameter>

```

Comment:
Can you increase this so that we don't have to keep doing this if we have to some more?